test/test_transforms_v2_refactored.py

```diff
@@ -2904,6 +2956,55 @@ def test__get_params(self, sigma):
             assert sigma[0] <= params["sigma"][0] <= sigma[1]
             assert sigma[0] <= params["sigma"][1] <= sigma[1]
 
+    # np_img = np.arange(3 * 10 * 12, dtype="uint8").reshape((10, 12, 3))
+    # np_img2 = np.arange(26 * 28, dtype="uint8").reshape((26, 28))
+    # {
+    #     "10_12_3__3_3_0.8": cv2.GaussianBlur(np_img, ksize=(3, 3), sigmaX=0.8),
+    #     "10_12_3__3_3_0.5": cv2.GaussianBlur(np_img, ksize=(3, 3), sigmaX=0.5),
+    #     "10_12_3__3_5_0.8": cv2.GaussianBlur(np_img, ksize=(3, 5), sigmaX=0.8),
+    #     "10_12_3__3_5_0.5": cv2.GaussianBlur(np_img, ksize=(3, 5), sigmaX=0.5),
+    #     "26_28_1__23_23_1.7": cv2.GaussianBlur(np_img2, ksize=(23, 23), sigmaX=1.7),
+    # }
+    REFERENCE_GAUSSIAN_BLUR_IMAGE_RESULTS = torch.load(
+        Path(__file__).parent / "assets" / "gaussian_blur_opencv_results.pt"
+    )
+
+    @pytest.mark.parametrize(
+        ("canvas_size", "kernel_size", "sigma"),
+        [
+            ("small", (3, 3), 0.8),
+            ("small", (3, 3), 0.5),
+            ("small", (3, 5), 0.8),
+            ("small", (3, 5), 0.5),
+            ("large", (23, 23), 1.7),
+        ],
+    )
+    @pytest.mark.parametrize("dtype", [torch.float32, torch.float64, torch.float16])
+    @pytest.mark.parametrize("device", cpu_and_cuda())
+    def test_functional_image_correctness(self, kernel_size, sigma, canvas_size, dtype, device):
```

Comment:
I've refactored this from

https://github.com/pytorch/vision/blob/4ac707a24a1705b384e1d73976dc0c075c6de5b6/test/test_functional_tensor.py#L1028

which is hard to understand and also does some unnecessary work just to skip later.